examples/batched-bench/batched-bench.cpp

```diff
@@ -37,8 +38,8 @@ static void print_usage(int, char ** argv) {
 int main(int argc, char ** argv) {
     gpt_params params;
 
-    auto options = gpt_params_parser_init(params, LLAMA_EXAMPLE_BENCH, print_usage);
-    if (!gpt_params_parse(argc, argv, params, options)) {
+    auto ctx_arg = gpt_params_parser_init(params, LLAMA_EXAMPLE_BENCH, print_usage);
+    if (!gpt_params_parse(argc, argv, ctx_arg)) {
         return 1;
     }
```

Comment:
This repeating pattern can be simplified with a wrapper:

```suggestion
    if (!gpt_params_parse_ext(argc, argv, params, LLAMA_EXAMPLE_BENCH, print_usage)) {
        return 1;
    }
```

Reply:
I ended up directly changing `gpt_params_parse` to `gpt_params_parse(argc, argv, params, LLAMA_EXAMPLE_BENCH, print_usage)`, so we have only one version of this function: https://github.com/ggerganov/llama.cpp/pull/9388/commits/96311e3248c9730b128b29999a4c8b7da0937ce9